components/PostCardList/PostCardListItemLoader.tsx

```diff
@@ -0,0 +1,176 @@
+import * as React from "react";
+import ContentLoader from "react-content-loader";
+import styled from "styled-components";
+import Card from "../Card";
+
+interface Props extends React.Attributes {
+  className?: string;
+  style?: React.CSSProperties;
+}
+
+export default function PostCardListItemLoader({ ...props }: Props) {
+  return (
+    <Card {...props}>
+      <Loader>
+        <HeroHand />
+
+        <TitleFirstLine />
+        <TitleSecondLine />
+        <TitleThirdLine />
+
+        <LikeIcon />
+        <LikeNumber />
+
+        <PlayAtTitle />
+        <EndedAtTitle />
+        <FinalPotTitle />
+        <PostedTitle />
+
+        <PlayAtValue />
+        <EndedAtValue />
+        <FinalPotValue />
+        <PostedValue />
+      </Loader>
+    </Card>
+  );
+}
+
+const Loader = styled(ContentLoader)`
+  --hero-hand-diameter: 96px;
```

Comment:
[Diameter](https://www.google.com/search?q=diameter) is not proper word to explain rectangles' length.